pkg/api/types.go

```diff
@@ -2262,6 +2262,11 @@ const (
 	// Number of Pods that may be running on this Node: see ResourcePods
 )
 
+const (
+	// Namespace prefix for opaque counted resources (alpha).
+	ResourceOpaqueIntPrefix = "opaque-int.alpha.kubernetes.io/"
```

Comment:
Searching the source code I found at least:

- pod.alpha.kubernetes.io
- pods.alpha.kubernetes.io
- scheduler.alpha.kubernetes.io
- security.alpha.kubernetes.io
- container.seccomp.security.alpha.kubernetes.io
- seccomp.security.alpha.kubernets.io
- create.snapshot.volume.alpha.kubernetes.io
- snapshot.volume.alpha.kubernetes.io
- container.apparmor.security.alpha.kubernetes.io
- metrics.alpha.kubernetes.io
- failure-domain.alpha.kubernetes.io
- service.alpha.kubernetes.io
- metadata.annotations.scheduler.alpha.kubernetes.io
- rkt.alpha.kubernetes.io

So... pod makes sense to me. Neither scheduler nor node capture the whole scope of where these values are used. Since it's more or less a free-for-all we could probably make the case for "resource" as a new prefix too (`opaque.resource.alpha.kubernetes.io`). Alternatively, we can match the existing alpha resource as mentioned above.